### PR TITLE
remove removeDropedPullRequestOpaque and deleteOpaqueForDropPullRequest.

### PR DESCRIPTION
**Pull request by ifplusor**
## What is the purpose of the change

clean up.

## Brief changelog

remove removeDropedPullRequestOpaque and deleteOpaqueForDropPullRequest.

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when a cross-module dependency exists.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Comment by jonnxu** [comment]
LGTM, good

File: src/MQClientAPIImpl.cpp [hunk 1]
```diff
@@ -430,10 +430,6 @@ void MQClientAPIImpl::sendMessageAsync(const string& addr,
   }
 }
 
-void MQClientAPIImpl::deleteOpaqueForDropPullRequest(const MQMessageQueue& mq, int opaque) {
```
Comment (jonnxu) [inline src/MQClientAPIImpl.cpp]: delete getLatestPullRequestOpaque and setLatestPullRequestOpaque the same time in this pr ?

Comment (ifplusor) [reply src/MQClientAPIImpl.cpp]: Yes, I have removed them.

File: src/MQClientFactory.cpp [hunk 2]
```diff
@@ -1012,10 +1000,6 @@ void MQClientFactory::resetOffset(const string& group,
       PullRequest* pullreq = pConsumer->getRebalance()->getPullRequest(mq);
       if (pullreq) {
         pullreq->setDroped(true);
-        LOG_INFO("resetOffset setDroped for opaque:%d, mq:%s", pullreq->getLatestPullRequestOpaque(),
```
Comment (jonnxu) [inline src/MQClientFactory.cpp]: i suggest reserve this log

File: src/MQClientAPIImpl.cpp [hunk 3]
```diff
@@ -467,21 +463,7 @@ void MQClientAPIImpl::pullMessageAsync(const string& addr,
...
-  AsyncArg* pAsyncArg = static_cast<AsyncArg*>(pArg);
-  if (pAsyncArg && pAsyncArg->pPullRequest) {
-    mq = pAsyncArg->mq;
-    pAsyncArg->pPullRequest->setLatestPullRequestOpaque(request.getOpaque());
-    LOG_DEBUG("pullMessageAsync set opaque:%d, mq:%s", pAsyncArg->pPullRequest->getLatestPullRequestOpaque(),
-              mq.toString().c_str());
-  }
-
   if (m_pRemotingClient->invokeAsync(addr, request, cbw, timeoutMillis) == false) {
-    LOG_ERROR("pullMessageAsync failed of addr:%s, opaque:%d, mq:%s", addr.c_str(), request.getOpaque(),
```
Comment (jonnxu) [inline src/MQClientAPIImpl.cpp]: i suggest reserve this log

**Comment by jonnxu** [comment]
LMGT

**Pull request merged**